controllers/dspipeline_params.go

```diff
@@ -483,16 +483,22 @@ func (p *DSPAParams) ExtractParams(ctx context.Context, dsp *dspa.DataSciencePip
 		APIServerArtifactImagePath := p.GetImageForComponent(dsp, config.APIServerArtifactImagePath, config.APIServerArtifactImagePathV2Argo, config.APIServerArtifactImagePathV2Tekton)
 		APIServerCacheImagePath := p.GetImageForComponent(dsp, config.APIServerCacheImagePath, config.APIServerCacheImagePathV2Argo, config.APIServerCacheImagePathV2Tekton)
 		APIServerMoveResultsImagePath := p.GetImageForComponent(dsp, config.APIServerMoveResultsImagePath, config.APIServerMoveResultsImagePathV2Argo, config.APIServerMoveResultsImagePathV2Tekton)
+		APIServerKFPLauncherImagePath := p.GetImageForComponent(dsp, config.APIServerKFPLauncherImagePath, config.APIServerKFPLauncherImagePathV2Argo, config.APIServerKFPLauncherImagePathV2Tekton)
+		APIServerDriverImagePath := p.GetImageForComponent(dsp, config.APIServerDriverImagePath, config.APIServerDriverImagePathV2Argo, config.APIServerDriverImagePathV2Tekton)
```

Comment:
we don't need to set the v2 tekton, and v1 parameters, we can just directly use
`config.APIServerDriverImagePathV2Argo` and `config.APIServerKFPLauncherImagePathV2Argo`, if you notice in the go template, these images are only utilized for v2 argo